docs/content/contribute/define-meeting/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OSCAL DEFINE Meeting
+description: Details about the OSCAL DEFINE Meeting
+layout: devlunch
+aliases:
+- /contribute/model-review/
+toc:
+  headingselectors: "h3,h4"
+  includeHtml: true
+weight: 20
+---
+
+{{% usa-intro %}}OSCAL DEFINE (**D**evelop **E**nhancements, **F**uture **I**mplementations and **N**ew **E**ducation) is a new meeting to discuss related to the research and educational pursuits of OSCAL using an iterative and collaborative approach with the community.  Our goal is to establish an OSCAL research framework that allows topics to receive the necessary definition and understanding sufficient to create educational content, demonstrate usage in practice, gain community’s endorsement, and serve as a catalyst for continued development of OSCAL.{{% /usa-intro %}}
+
+{{% usa-tag %}}Day and Time{{% /usa-tag %}} Monthly Thursday at 10am EST/EDT
```

Comment:
This will conflict with the model engineering meeting every other month or so. Shouldn't this be at 11am to avoid this?

Reply:
Yes, the meeting was discussed to be held on the third Th of each month at 11:00AM ET. @Compton-NIST - do you want the meeting at 10?

Reply:
Updated accordingly.